makefile

```diff
@@ -159,6 +159,14 @@ gen-fixtures:
 	$(call log, "Generating new article fixture data")
 	@node scripts/test-data/gen-fixtures.js
 
+gen-fixtures-and-schema:
+	$(call log, "Generating new article fixture data and new schema")
+	make gen-schema && make gen-fixtures
+
+update-manual-fixtures:
+	$(call log, "Updating all manual fixture data")
+	@node scripts/test-data/update-manual-fixtures.js
+
```

Comment:
I think this represents my issue here. Manual fixutes are manual because they are not updated by scripts. That's the whole point of the `manual` folder. But what we're doing here is we're explicitly breaking that model by creating a automatied script to generate 'manual' data

Reply:
I'd argue that the distinction is that we're producing generated fixtures from manual data. Whereas the other script generates fixtures from automated data